### PR TITLE
Version 2 sim release

### PR DESCRIPTION
**Pull request by antoinepetty**

### CODE REVIEW

**Comment by antoinepetty** [comment]
Do we want to announce an official date?

File: SR2020/2020-06-18-version-0-2-0-simulator.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Hi
+
+We have released version 0.2.0 of the competition simulation which you can download from the [Docs page](https://studentrobotics.org/docs/competition-simulator/).
+
+The following changes have been made:
+
+- Reducing the range of randomness which can be exhibited by motors and sensors
+- Adding multiple robots to the arena with the ability to assign distinct code files to each
+- Fix the gripper jumping open bug
+- Add a sleep method which runs on "simulation time" instead of actual time. This is useful for computers where the simulation runs slowly
```
Comment (PeterJCLaw) [inline SR2020/2020-06-18-version-0-2-0-simulator.md:15]: Have we actually documented this anywhere? If not, that's a blocker for release.

File: SR2020/2020-06-18-version-0-2-0-simulator.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+to: sr2020-teams
+subject: Virtual Competition Simulation Version 2 Released
+---
+
+Hi
+
+We have released version 0.2.0 of the competition simulation which you can download from the [Docs page](https://studentrobotics.org/docs/competition-simulator/).
```
Comment (rgilton) [inline SR2020/2020-06-18-version-0-2-0-simulator.md:8]: Are the forms not a better place for this announcement?

File: SR2020/2020-06-18-version-0-2-0-simulator.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2020/2020-06-18-version-0-2-0-simulator.md:15]: Tracking as https://github.com/srobo/competition-simulator/issues/141

File: SR2020/2020-06-18-version-0-2-0-simulator.md (hunk 2 above)
Comment (RealOrangeOne) [reply SR2020/2020-06-18-version-0-2-0-simulator.md:8]: It'll be announced there too. The issue is that the forums aren't read as regularly by some teams, so an email deals with both ensuring everyone gets the message, and prompting those who haven't signed up yet.

File: SR2020/2020-06-18-version-0-2-0-simulator.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The following changes have been made:
```
Comment (RealOrangeOne) [inline SR2020/2020-06-18-version-0-2-0-simulator.md:10]: There are technically some other changes they may notice, but I think these are the ones they really care about

```suggestion
We've made a number of changes, most notably:
```

File: SR2020/2020-06-18-version-0-2-0-simulator.md (hunk 2 above)
Comment (rgilton) [reply SR2020/2020-06-18-version-0-2-0-simulator.md:8]: Sorry, I totally missed that this was an email going out to teams, not volunteers.  Ignore me.

File: SR2020/2020-06-18-version-0-2-0-simulator.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+to: sr2020-teams
+subject: Virtual Competition Simulation Version 2 Released
```
Comment (trickeydan) [inline SR2020/2020-06-18-version-0-2-0-simulator.md:3]:
```suggestion
subject: Virtual Competition Simulator Update
```

Need to keep email subjects short for some mail clients.

**Pull request merged**